doc/classes/Mesh.xml

```diff
@@ -366,6 +366,9 @@
 		<constant name="ARRAY_FLAG_USES_EMPTY_VERTEX_ARRAY" value="268435456" enum="ArrayFormat" is_bitfield="true">
 			Flag used to mark that the mesh intentionally contains no vertex array.
 		</constant>
+		<constant name="ARRAY_FLAG_COMPRESS_ATTRIBUTES" value="536870912" enum="ArrayFormat" is_bitfield="true">
+			Flag used to mark that a mesh is using compressed attributes (vertices, normals, tangents, uvs). When this form of compression is enabled, vertex positions will be packed into into an RGBA16UNORM attribute and scaled in the vertex shader. The normal and tangent will be packed into a RG16UNORM representing an axis, and an 16 bit float stored in the A-channel of the vertex. UVs will use 16-bit normalized floats instead of full 32 bit signed floats. When using this compression mode you must either use vertices, normals, and tangents or only vertices. You cannot use normals without tangents. Importers will automatically enable this compression if they can.
```

Comment:
Seems like we try to use `UVs` instead of `uvs` in the docs. Actually this description uses both.
```
$ rg -g'*.xml' '[^"]UVs[^"]' | wc -l
15
$ rg -g'*.xml' '[^"]uvs[^"]' | wc -l
2
$ rg -g'*.xml' '[^"]uvs[^"]'
doc/classes/ImporterMesh.xml
95:                             Returns the arrays for the vertices, normals, uvs, etc. that make up the requested surface. See [method add_surface].

doc/classes/RenderingServer.xml
4229:                   Flag used to mark that a mesh is using compressed attributes (vertices, normals, tangents, uvs). When this form of compression is enabled, vertex positions will be packed into into an RGBA16UNORM attribute and scaled in the vertex shader. The normal and tangent will be packed into a RG16UNORM representing an axis, and an 16 bit float stored in the A-channel of the vertex. UVs will use 16-bit normalized floats instead of full 32 bit signed floats. When using this compression mode you must either use vertices, normals, and tangents or only vertices. You cannot use normals without tangents. Importers will automatically enable this compression if they can.
```

I see it's copied from RS so the fixes should be done there too.
Note: Lots of typo/language fixes.

```suggestion
			Flag used to mark that a mesh is using compressed attributes (vertices, normals, tangents, UVs). When this form of compression is enabled, vertex positions will be packed into an RGBA16UNORM attribute and scaled in the vertex shader. The normal and tangent will be packed into an RG16UNORM representing an axis, and a 16-bit float stored in the A-channel of the vertex. UVs will use 16-bit normalized floats instead of full 32-bit signed floats. When using this compression mode you must either use vertices, normals, and tangents, or only vertices. You cannot use normals without tangents. Importers will automatically enable this compression if they can.
```

Reply:
Updated, indeed the last comma isn't correct. You shouldn't use a a comma to separate an either/or statement. I think part of what made that confusing was that "use" came after the "either" but "use" applied to both sides of the either/or. 

I rearranged it to say "...you must use either vertices, normals, and tangents or only vertices." instead